packages/core/package.json

```diff
@@ -140,7 +140,9 @@
     "dev": "tsdown --watch",
     "test": "vitest run",
     "test:ui": "vitest --ui",
-    "prepublishOnly": "cp ../../README.md ./README.md"
+    "prepublishOnly": "cp ../../README.md ./README.md",
+    "lint": "eslint .",
```

Comment:
so simplify, just start with core package first and if necessary deactivate some rules in eslint.config